tests/integration/api_admin_test.go

```diff
@@ -77,15 +78,15 @@ func TestAPIAdminDeleteUnauthorizedKey(t *testing.T) {
 	token = getTokenForLoggedInUser(t, session)
 	req = NewRequestf(t, "DELETE", "/api/v1/admin/users/%s/keys/%d?token=%s",
 		adminUsername, newPublicKey.ID, token)
-	session.MakeRequest(t, req, http.StatusForbidden)
+	session.MakeRequest(t, req, http.StatusUnauthorized)
```

Comment:
I think `Unauthorized` means your token is not right but `Forbidden` means you have no enough permission to visit this resource.